www/data/docs_sidebar.yml

```diff
@@ -96,6 +93,23 @@ sidebar_links:
       link: "/docs/reference/#sup-log-keys"
     - title: Infographics
       link: "/docs/reference/#infographics"
+  - title: Best Practice Guides
+    link: "/docs/best-practices"
+    sub_links:
+    - title: Container Orchestration
+      link: "/docs/best-practices/#container-orchestration"
+    - title: Kubernetes
+      link: "/docs/best-practices/#kubernetes"
+    - title: "ECS and Habitat"
+      link: "/docs/best-practices/#ecs-and-habitat"
+    - title: "Apache Mesos and DC/OS"
+      link: "/docs/best-practices/#mesos-dcos"
+    - title: "Advanced Plan.sh Writing"
```

Comment:
Also here.

Reply:
Are you referring to the missing page title on the Best Practices page?

Reply:
Oh sorry, I just meant to suggest a rename to "Advanced Plan Writing" if you agreed with my earlier suggestion.